indra_reading/readers/trips/__init__.py

```diff
@@ -60,9 +72,12 @@ def _start_trips():
         service_host = 'http://localhost:%d/cgi/' % port
```

Comment:
Under `if os.environ.get("IN_TRIPS_DOCKER", 'false') == 'true':` set service_port = 80, then under `else` set service_port = port. Then in line 60 set 
```
service_host = 'http://localhost:%d/cgi/' % service_port
```